AspNet.Security.OAuth.Providers.sln.DotSettings

```diff
@@ -21,4 +21,8 @@
     <s:Boolean x:Key="/Default/CodeStyle/CodeFormatting/CSharpFormat/SPACE_BEFORE_TYPEOF_PARENTHESES/@EntryValue">False</s:Boolean>
     <s:String x:Key="/Default/CodeStyle/CodeFormatting/CSharpFormat/TYPE_DECLARATION_BRACES/@EntryValue">END_OF_LINE</s:String>
     <s:String x:Key="/Default/CodeStyle/Naming/CSharpNaming/PredefinedNamingRules/=PrivateInstanceFields/@EntryIndexedValue">&lt;Policy Inspect="True" Prefix="" Suffix="" Style="aaBb" /&gt;</s:String>
-    <s:String x:Key="/Default/CodeStyle/Naming/CSharpNaming/PredefinedNamingRules/=PrivateStaticFields/@EntryIndexedValue">&lt;Policy Inspect="True" Prefix="" Suffix="" Style="aaBb" /&gt;</s:String></wpf:ResourceDictionary>
\ No newline at end of file
+    <s:String x:Key="/Default/CodeStyle/Naming/CSharpNaming/PredefinedNamingRules/=PrivateStaticFields/@EntryIndexedValue">&lt;Policy Inspect="True" Prefix="" Suffix="" Style="aaBb" /&gt;</s:String>
```

Comment:
Don't worry about messing everything up: if things go really bad locally, just run `git reset --hard origin/linkedin` and you'll get back the current history of your fork on GitHub :smile: 

`git rebase -i` is rather easy to use, as it's graphical and uses your favorite text editor to edit the git history. When running `git rebase -i HEAD~5` (that takes the last 5 commits), select the line corresponding to the 2 commits you want to remove and delete them. Then, save the file and close your text editor. If you're happy with the changes (don't hesitate to use GitHub client's graphical history for that), then, just force push: `git push -f`.

Anyway, if you're afraid of doing bad things, I'll fix everything when merging :+1: